Sources/StreamChatSwiftUI/ChatChannel/ChatChannelView.swift

```diff
@@ -163,13 +161,6 @@ public struct ChatChannelView<Factory: ViewFactory>: View, KeyboardReadable {
         .onDisappear {
             viewModel.onViewDissappear()
         }
-        .onChange(of: presentationMode.wrappedValue, perform: { newValue in
-            if newValue.isPresented == false {
-                viewModel.onViewDissappear()
-            } else {
-                viewModel.setActive()
-            }
-        })
```

Comment:
This doesn't work.
I assume it has the same issue as `scenePhase`: https://stackoverflow.com/q/62840571